embulk-cli/src/main/java/org/embulk/cli/EmbulkMigrate.java

```diff
@@ -292,19 +292,23 @@ public FileVisitResult visitFile(Path filePath, BasicFileAttributes attributes)
                             int position = 0;
                             String modifiedData = originalData;
                             while (position < modifiedData.length()) {
-                                final Matcher matcher = pattern.matcher(modifiedData.substring(position));
-                                if (!matcher.matches()) {
+                                final String postModifiedData = modifiedData.substring(0,position);
+                                final String preModifiedData = modifiedData.substring(position);
```

Comment:
1. `postModifiedData` sounds like "the data **after** `modifiedData`", but the code actually looks like "the earlier part of `modifiedData`". Which one did you intend? If it means like "the earlier part", I'd suggest like:
    * `formerModifiedData` / `latterModifiedData`
    * `earlierModifiedData` / `laterModifiedData`
    * `modifiedDataBeforePosition` / `modifiedDataAfterPosition`
2. Anyway even if we use this `post-` and `pre-`, they sound reversed. `post-` is a kind of "after something", and `pre-` is a kind of "before something".